src/ImageSharp/Processing/Quantization/QuantizationMode.cs

```diff
@@ -4,26 +4,26 @@
 namespace SixLabors.ImageSharp.Processing.Quantization
 {
     /// <summary>
-    /// Provides enumeration over how an image should be quantized.
+    /// Contains reusable static instances of known quantizing algorithms
     /// </summary>
-    public enum QuantizationMode
+    public static class QuantizationMode
```

Comment:
:+1: for dropping the enum! I think we should also change the name to `KnownQuantizers` to match our typical convention.

Reply:
We don't use the `KnownXXX` anymore within the API, I dropped that with the namespace refactor since it didn't match the normal MS convention.

Reply:
I see. I guess  we do not use simple plural for these classes to avoid collision with namespaces?
`XXXMode` intuitively suggests a closed API to me. Is this the standard MS convention now?

Reply:
> `XXXMode` intuitively suggests a closed API to me. 

Yeah, you're right, it does doesn't it. Mode isn't a standard suffix, the naming is usually something that would clash horribly with our namespaces.

I'm happy to change all our static aggregators to use the prefix `Known` again as it does open up the API.

Reply:
To me it looks like the `Known` prefix is the least worst of all those options. Any recommendation from MS?

Reply:
I'll ask Immo